spring-cloud-commons/src/main/java/org/springframework/cloud/client/discovery/composite/CompositeDiscoveryClient.java

```diff
@@ -73,6 +73,15 @@ public List<String> getServices() {
 		return new ArrayList<>(services);
 	}
 
+	@Override
```

Comment:
Please add your name with `@author` tag in the javadoc of the edited file. Please update the copyright date line.